spec/features/manage_cmr/subscriptions/test_query_spec.rb

```diff
@@ -0,0 +1,306 @@
+    @ingest_subscription_response, @search_response, _subscription = publish_new_subscription(collection_concept_id: @ingest_response['concept-id'])
+    VCR.use_cassette('urs/rarxd5taqea', record: :none) do
+      visit edit_subscription_path(@ingest_subscription_response['concept_id'])
+    end
+  end
+
+  after do
+    delete_response = cmr_client.delete_subscription('MMT_2', @search_response.body['items'].first['meta']['native-id'], 'token')
+
+    raise unless delete_response.success?
+  end
+
+  context 'when the form is incomplete' do
+    before do
+      fill_in 'Query', with: ''
+      click_on 'Estimate Notification Frequency'
+    end
+
+    it 'validates the form' do
+      expect(page).to have_content('Query is required.')
+    end
+
+    it 'displays a modal with an informative message' do
+      expect(page).to have_content('Please enter a valid subscription and try again.')
+    end
+  end
+
+  context 'when the form is complete' do
+    before do
+      fill_in 'Query', with: 'day_night_flag=day&updated_since=2020-05-06T16:23:09Z&production_date[]=2000-01-01T10:00:00Z,2021-03-10T12:00:00Z'
+    end
+
+    context 'when the query starts with a "?"' do
+      before do
+        @granules_in_test = 40
+        ingest_granules(@concept_response.body['EntryTitle'], @granules_in_test, 'MMT_2')
+        wait_for_cmr
+        fill_in 'Query', with: '?day_night_flag=day&updated_since=2020-05-06T16:23:09Z&production_date[]=2000-01-01T10:00:00Z,2021-03-10T12:00:00Z'
+        click_on 'Estimate Notification Frequency'
+      end
+
+      it 'can succeed' do
+        expect(page).to have_content('Estimate Done')
+        expect(page).to have_content("Estimate: #{(@granules_in_test / 30.0).ceil} notifications/day")
+        expect(page).to have_content('How was this estimate generated?')
+        expect(page).to have_content("#{@granules_in_test} granules related to this query were added or updated over the last 30 days.")
+        expect(page).to have_content("Notification service checks for new or updated granules once every #{@frequency}.")
+        expect(page).to have_content('Estimate assumes an even distribution of new or updated granules over a period of time.')
+      end
+    end
+
+    context 'when there is an error communicating with CMR' do
+      before do
+        cmr_response = Cmr::Response.new(Faraday::Response.new(status: 401, body: JSON.parse('{"errors": ["One of parameters [user_type] or [user_id] are required."]}'), response_headers: {}))
+        allow_any_instance_of(Cmr::CmrClient).to receive(:check_user_permissions).and_return(cmr_response)
+        click_on 'Estimate Notification Frequency'
+      end
+
+      it 'displays an error message on the modal' do
+        expect(page).to have_content("An error occurred while checking the user's permissions: One of parameters [user_type] or [user_id] are required.")
+      end
+    end
+
+    context 'when the test succeeds' do
```

Comment:
You had mentioned that the estimation tests for update were somewhat duplicative. I agree that you should have some tests, but I don't think you need all of the duplicative ones. Happy to discuss this further but I think you should keep:
- 'when the form is incomplete'
- _one_ of the tests under 'when the test succeeds'
I don't think you need to duplicate the others for the update form estimation.